### PR TITLE
Defines .mtr as material and sets it to display as plain text

### PR DESCRIPTION
**Pull request by shadguy**
Defines files with .mtr extension as materials and sets them up to display as plaintext.   This is an EE-specific asset type, goes hand in hand with shaders.

### CODE REVIEW

File: CppProperties.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
+{
```
Comment (virusman) [inline CppProperties.json]: Looks like these two files are not related to the PR.

Comment (shadguy) [reply CppProperties.json]: Thank you, fixed.  FYI, the changes to the fmod include in two files were needed to build and test on my machine, at least for now.  I'm not sure they're required for someone more experienced.

File: nwnexplorer/SoundView.h [hunk 2]
```diff
@@ -50,7 +50,7 @@
 #include "resource.h"
 #include "DataRef.h"
 #include "CommonView.h"
-#include <fmod.h>
+#include "include\fmod\fmod.h"
```
Comment (virusman) [inline nwnexplorer/SoundView.h]: Can you remove the fmod include changes? I think that'd be better for a separate PR. The code compiles as is on my machine (not sure how), and I think it may be better to fix the default include path in project settings instead.

**Comment by shadguy** [comment]
Done, thanks.  I've removed that for now, didn't figure out config yet.  
Also committed:

- version bump to 1.8.4 from 1.8.3
- changed "Material" folder name to "Materials" for more consistent grammar for end users
- cleaned up some unrelated ideas that leaked in here during fmod cleanup and the above work.  

Note: Untested, doesn't compile for me without the fmod changes.

**Comment by virusman** [comment]
Thank you!
